### ISSUE TITLE
Focus on first invalid field after closing Errors/Validation dialog

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#5883]
We have a built-in `validate-all` process, that calls `review-messages`, which does:

```xml
<property as="xs:string"  name="oxf.fr.detail.process.review-messages.*.*">
    validate("warning")
    then validate("info")
    recover (xf:show("fr-validation-dialog") then suspend)
</property>
```

The validation dialog is implemented in `validation-dialog.xhtml`. On close, the dialog runs the `validation-dialog-review` process, which does just `abort`. Before the `abort`, we could dispatch an event to the error summary, asking it to set the focus on the first invalid field. The error summary could do it as it has the logic we need to navigate to errors.

[+1 from customer](https://3.basecamp.com/3600924/buckets/1956846/messages/6307235213)

**Comment by ebruchez** [comment]
Should be easy to do based on the above.

**Comment by ebruchez** [comment]
There are two dialogs:

- Error Dialog
    - tells the user for `error` levels
- Validation Dialog
    - tells the user for `warning` and `info` levels

This happens in sequence:

```xml
<property as="xs:string"  name="oxf.fr.detail.process.validate-all.*.*">
    require-valid
    then review-messages
</property>
```

**Comment by ebruchez** [comment]
Either:

- we make both use the Validation Dialog
- we allow the Error Dialog to `resume` a process
   - problem: it's used by processes, but not always, so we don't know if we should resume ir not

**Comment by ebruchez** [comment]
I don't know why we don't use the Validation Dialog for regular errors, but we should.

**Comment by avernet** [comment]
[+1 from customer](https://3.basecamp.com/3600924/buckets/1966718/messages/6338049163)
